docker/Dockerfile

```diff
@@ -10,38 +29,20 @@ ARG PROJECT_VERSION=2.x
 # Set Apache ServerName directive globally to suppress AH00558 message.
 RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
 
+# Install and enable geos
+COPY --from=geos /usr/local/lib/php/extensions/ /usr/local/lib/php/extensions/
+RUN docker-php-ext-enable geos
+
 # Install the BCMath PHP extension.
 RUN docker-php-ext-install bcmath
 
-# Install git and unzip (needed to build PHP GEOS and needed by Composer).
-RUN apt-get update \
-  && apt-get install -y git unzip
-
-# Build and install the GEOS PHP extension.
-# See https://git.osgeo.org/gitea/geos/php-geos
-RUN apt-get update && apt-get install -y libgeos-dev \
-  && git clone https://github.com/libgeos/php-geos.git \
-  && ( \
-    cd php-geos \
-    # Checkout latest commit with PHP 8 support.
-    && git checkout e77d5a16abbf89a59d947d1fe49381a944762c9d \
-    && ./autogen.sh \
-    && ./configure \
-    && make \
-    && make install \
-  ) \
-  && rm -r php-geos \
-  && docker-php-ext-enable geos
-
 #Add custom configurations
 COPY conf.d/ /usr/local/etc/php/conf.d
 
-# Install postgresql-client so Drush can connect to the database.
 RUN apt-get update \
-  # See https://stackoverflow.com/questions/51033689/how-to-fix-error-on-postgres-install-ubuntu
```

Comment:
Is this no longer an issue? This should be a separate change/commit with explanation as to why we can remove these lines.